portfolio/src/main/webapp/comment.html

```diff
@@ -5,16 +5,26 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="https://kit.fontawesome.com/e073fdea10.js" crossorigin="anonymous"></script>
-    <script src="script.js"></script>
+    <script src="comment-form.js"></script>
   </head>
-  <body class="container comment-container">
+  <body class="container comment-container" onload="getComments()">
     <nav class="navbar">
       <a href="comment.html">comment</a>
       <a href="contact.html">contact</a>
       <a href="projects.html">projects</a>
       <a href="index.html">about</a>
     </nav>
-    <section class="container comment-section-container"></section>
+    <main class="container comment-section-container">
+      <section id="comment-form" class="container form-container">
+        <form action="/handle-comment" method="POST">
+          <label for="user-comment">Leave a comment below:</label><br>
```

Comment:
You can make labels and inputs `display: block` So that each element gets a newline, instead of using `<br>`. Also `display: flex` and `flex-direction: column` will accomplish this.

Try to avoid using `<br>` to make whitespace and newlines, CSS can handle this much more cleanly.

Reply:
fixed